ui/apps/platform/src/css/trumps.css

```diff
@@ -135,6 +135,10 @@
     display: inline;
 }
 
+.pf-c-progress__status-icon svg {
+  display: inline-block;
+}
+
```

Comment:
@bradr5 It appears that Dave needs a PF override for svg to `inline-block` as well, like you do in Compliance 2.0

We should eventually merge your two selectors into the same rule.